x-pack/test/apm_api_integration/tests/alerts/anomaly_alert.spec.ts

```diff
@@ -15,29 +15,30 @@ import { waitForRuleStatus } from './wait_for_rule_status';
 
 export default function ApiTest({ getService }: FtrProviderContext) {
   const registry = getService('registry');
-  const synthtraceEsClient = getService('synthtraceEsClient');
-  const ml = getService('ml');
+
   const supertest = getService('supertest');
+  const ml = getService('ml');
   const log = getService('log');
 
-  // FLAKY: https://github.com/elastic/kibana/issues/129224
-  registry.when.skip(
+  const synthtraceEsClient = getService('synthtraceEsClient');
+
+  registry.when(
     'fetching service anomalies with a trial license',
     { config: 'trial', archives: ['apm_mappings_only_8.0.0'] },
     () => {
-      const spikeStart = datemath.parse('now-2h')!.valueOf();
-      const spikeEnd = datemath.parse('now')!.valueOf();
+      const start = '2021-01-01T00:00:00.000Z';
+      const end = '2021-01-08T00:15:00.000Z';
 
-      const start = datemath.parse('now-2w')!.valueOf();
-      const end = datemath.parse('now')!.valueOf();
+      const spikeStart = new Date('2021-01-07T20:00:00.000Z').getTime();
```

Comment:
Can we make the spike a little shorter? If the spike takes too long the ML job will adjust its anomaly score because it gets "used" to the spike. Like an hour?